projects/cfengine/string_fuzzer.c

```diff
@@ -0,0 +1,28 @@
+/* Copyright 2021 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <stdint.h>
+#include <string.h>
+#include <stdlib.h>
+#include <string_expressions.h>
+
+int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size){
```

Comment:
How would we add fuzzing of more functions in this setup?

Reply:
A single file can maximum have one LLVMFuzzerTestOneInput api in it. There are several options to add more fuzzers, of which one is to have several separate files, or if the input is similar we could use a single LLVMFuzzerTestOneInput to call into several targets.

Reply:
Thanks for the answer, make ssense.